src/Microsoft.AspNet.Server.Kestrel/project.json

```diff
@@ -52,13 +52,11 @@
     "keyFile": "../../tools/Key.snk"
   },
   "scripts": {
-    "prepare": [
-      "dnu restore ../../tools/Microsoft.AspNet.Server.Kestrel.GeneratedCode",
-      "dnx -p ../../tools/Microsoft.AspNet.Server.Kestrel.GeneratedCode run Http/FrameHeaders.Generated.cs Http/Frame.Generated.cs"
-    ],
-    "postrestore": [
-      "dnu restore ../../tools/Microsoft.AspNet.Server.Kestrel.LibuvCopier",
-      "dnx -p ../../tools/Microsoft.AspNet.Server.Kestrel.LibuvCopier run"
+    "precompile": [
+      "dotnet restore ../../tools/Microsoft.AspNet.Server.Kestrel.GeneratedCode",
+      "dotnet run -p ../../tools/Microsoft.AspNet.Server.Kestrel.GeneratedCode Http/FrameHeaders.Generated.cs Http/Frame.Generated.cs",
+      "dotnet restore ../../tools/Microsoft.AspNet.Server.Kestrel.LibuvCopier",
+      "dotnet run -p ../../tools/Microsoft.AspNet.Server.Kestrel.LibuvCopier"
```

Comment:
dotnet is a native executable. doesn't require any extension and it's only coreclr